x-pack/plugins/ml/public/application/data_frame_analytics/pages/analytics_exploration/components/total_feature_importane_summary/feature_importance_summary.tsx

```diff
@@ -0,0 +1,208 @@
+      if (totalFeatureImportance[0].classes.length > 2) {
+        (totalFeatureImportance as ClassificationTotalFeatureImportance[]).forEach((feature) => {
+          feature.classes
+            .sort((a, b) => a.importance.mean_magnitude - b.importance.mean_magnitude)
+            .forEach((featureClass) => {
+              sortedData.push({
+                featureName: feature.feature_name,
+                meanImportance: featureClass.importance.mean_magnitude,
+                className: featureClass.class_name,
+              });
+            });
+        });
+        barSettings = {
+          xAccessor: 'featureName',
+          yAccessors: ['meanImportance'],
+          splitSeriesAccessors: ['className'],
+          stackAccessors: ['featureName'],
+        };
+      }
+    }
+    // if regression
+    if (isRegressionTotalFeatureImportance(totalFeatureImportance[0])) {
+      sortedData = (totalFeatureImportance as RegressionTotalFeatureImportance[]).map(
+        (d: RegressionTotalFeatureImportance) => ({
+          featureName: d.feature_name,
+          meanImportance: d.importance.mean_magnitude,
+        })
+      );
+    }
+
+    // sort from largest importance at top to smallest importance at bottom
+    sortedData = sortedData.sort((a, b) => b.meanImportance - a.meanImportance);
+
+    return [sortedData, barSettings];
+  }, [totalFeatureImportance]);
+
+  const { ELASTIC_WEBSITE_URL, DOC_LINK_VERSION } = docLinks;
+  const tickFormatter = useCallback((d) => Number(d.toPrecision(3)).toString(), []);
+
+  return (
+    <EuiPanel>
+      <EuiFlexGroup>
+        <EuiFlexItem grow={false}>
+          <EuiTitle size="xs">
+            <span>
+              <FormattedMessage
+                id="xpack.ml.dataframe.analytics.exploration.featureImportanceSummaryTitle"
+                defaultMessage="Feature importance summary"
+              />
+            </span>
+          </EuiTitle>
+        </EuiFlexItem>
+        <EuiFlexItem grow={false}>
+          <EuiIconTip content={tooltipContent} />
+        </EuiFlexItem>
+        <EuiFlexItem>
+          <EuiSpacer />
+        </EuiFlexItem>
+        <EuiFlexItem grow={false}>
+          <EuiButtonEmpty
+            target="_blank"
+            iconType="help"
+            iconSide="left"
+            color="primary"
+            href={`${ELASTIC_WEBSITE_URL}guide/en/machine-learning/${DOC_LINK_VERSION}/ml-feature-importance.html`}
+          >
+            <FormattedMessage
+              id="xpack.ml.dataframe.analytics.exploration.featureImportanceDocsLink"
+              defaultMessage="Feature importance docs"
+            />
+          </EuiButtonEmpty>
+        </EuiFlexItem>
+      </EuiFlexGroup>
+      <Chart className="story-chart" size={{ width: '100%', height: 300 }}>
+        <Settings rotation={90} theme={theme} showLegend />
+
+        <Axis
+          id="x-axis"
+          title="Feature importance average magnitude"
```

Comment:
This needs to be internationalized if possible.

Reply:
Updated here https://github.com/elastic/kibana/pull/78238/commits/c419c4223cb0609006dcae4b37197becc49b2176